source/api_documentation/network_integration/tts_prompt_voice/index.rst

```diff
@@ -0,0 +1,213 @@
+TTS Prompt Voice
+======================
+
+For full details on creating and updating campaigns via the API, see :doc:`Advertiser Campaigns <../advertiser_campaigns/index>`.
+
+Manage TTS Prompt Voice for Campaigns
+"""""""""""""""""""""""""""""""""""""""""""
+
+TTS prompt voice can be sent for a campaign to use different accents for TTS prompts on a call.
+
+Set TTS Prompt Voice
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To generate voice prompts using a certain prompt voice for a given campaign, set ``"tts_voice_talent_id"`` on the IVR you want.
+
+POST
+
+``https://invoca.net/api/@@CAMPAIGN_FEATURES_API_VERSION/<network_id>/advertisers/<advertiser_id_from_network>/advertiser_campaigns/<advertiser_campaign_id_from_network>.json``
+
+Example Request Body
+
+.. code-block:: json
+
+  {
+    "ivr_tree": {
+      "tts_voice_talent_id": 3,
+      "root": {
+        "node_type": "Connect",
```

Comment:
Since we are using `tts_voice_talent_id`, it could be helpful to include a `prompt` field in the example.

Reply:
Good point. Done.